### DISCUSSION TITLE
Liferea not updating updated posts

### DISCUSSION THREAD

**Discussion by reidrac** [lwindolf/liferea#1358] [Q&A]
I'm using Liferea 1.14.4 as packaged by Debian.

I have noticed that when a blog post is updated, Liferea seems to do the feed update (I don't see the "... has not changed since the last update" in the status bar on the bottom). But **the entries on the feed are the same**, not showing the updated content.

When I go to the website or I inspect the XML (for example RSS 2.0), I see the updated content. I can remove the feed and add it again to Liferea, and it shows correctly.

Is this is some strange behaviour in Liferea or I happen to have several feeds that are "wrong" missing something on them?

For example https://www.usebox.net/jjm/blog/index.xml has the first post (title "Funco") published on April 22nd and updated today 29th.

**Comment by lwindolf** [comment]
Missed this here. Rarely look at new discussions.

Can you create an issue? I'll look into it.
